packages/flutter_app/ios/Runner/Info.plist

```diff
@@ -80,5 +80,6 @@
 		<true/>
 		<key>UIApplicationSupportsIndirectInputEvents</key>
 		<true/>
-	</dict>
+		<key>kMDItemKeywords</key>
+		<string>fab, flutter_app_template,ふらったー,てんぷれーと</string>	</dict>
```

Comment:
```suggestion
		<string>fab, flutter_app_template,ふらったー,てんぷれーと</string>	
	</dict>
```

It is not that big of an impact, but I would like to clean up the formatting, including line breaks.

Reply:
For some reason the formatter didn't seem to be working!Thanks!